lib/beaker/ssh_connection.rb

```diff
@@ -69,8 +69,15 @@ def connect_block host, user, ssh_opts, options
       begin
          @logger.debug "Attempting ssh connection to #{host}, user: #{user}, opts: #{ssh_opts}"
 
+         # Work around net-ssh 6+ incompatibilities
          if ssh_opts.include?(:strict_host_key_checking) && (Net::SSH::Version::CURRENT.major > 5)
-           ssh_opts[:paranoid] = ssh_opts.delete(:strict_host_key_checking)
+           strict_host_key_checking = ssh_opts.delete(:strict_host_key_checking)
+
+           if ssh_opts[:verify_host_key].nil?
+             ssh_opts[:verify_host_key] = strict_host_key_checking ? :always : :never
+           else
+             ssh_opts[:verify_host_key] = (ssh_opts[:verify_host_key] ? :always : :never) unless ssh_opts[:verify_host_key].is_a?(Symbol)
+           end
```

Comment:
I believe this should do the same thing in a more succinct manner?
```suggestion
           ssh_opts[:verify_host_key] ||= strict_host_key_checking ? :always : :never
```

Reply:
@highb The value needs to be preserved if it's already a `Symbol`

Reply:
```suggestion
           unless ssh_opts[:verify_host_key].is_a?(Symbol)
             ssh_opts[:verify_host_key] ||= strict_host_key_checking ? :always : :never
           end
```
How about this?

Reply:
@highb Seems reasonable to me.

Reply:
`nil` has `#is_a?` so it should work:
```
[1] pry(main)> foo = nil
=> nil
[2] pry(main)> foo.is_a?(Symbol)
=> false
[3] pry(main)> foo = :banana
=> :banana
[4] pry(main)> foo.is_a?(Symbol)
=> true
```